src/library-authoring/utils.ts

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
nit: Could you move this code to [src/generic/key-utils.ts](https://github.com/openedx/frontend-app-authoring/blob/master/src/generic/key-utils.ts) (and add a test)?